common/src/main/java/com/genexus/ModelContext.java

```diff
@@ -43,7 +43,9 @@ public static ModelContext getModelContext()
         ModelContext context = (ModelContext)threadModelContext.get();
 		if(context == null)
 		{
-			logger.error(new Date() + " - Cannot find ModelContext for thread " + Thread.currentThread() );
+			if (logger.isDebugEnabled()) {
```

Comment:
It's not necessary to ask for isDebugEnabled.

Reply:
The debug method already have the if statement

	public void debug(String msg) {
		if (log.isDebugEnabled()) {
			log.debug(msg);
		}
	}

I understand the performance issue, but it seems that is a redundant code

Reply:
Ready, removed "if (log.isDebugEnabled()) {" from "public void debug(String msg)" method.